src/background/hypothesis-chrome-extension.js

```diff
@@ -31,24 +32,29 @@ const TAB_STATUS_COMPLETE = 'complete';
  * - https://developer.chrome.com/extensions/tabs
  * - https://developer.chrome.com/extensions/extension
  *
- * dependencies - An object to set up the application.
- *   chromeTabs: An instance of chrome.tabs.
- *   chromeBrowserAction: An instance of chrome.browserAction.
- *   extensionURL: chrome.extension.getURL.
- *   isAllowedFileSchemeAccess: chrome.extension.isAllowedFileSchemeAccess.
+ * @param {Object} services
```

Comment:
Thanks for the suggestion. In this context it doesn't appear to make any functional difference - if you try to pass a primitive or null/undefined it will error. I've stuck with `Object` as that's more conventional in our code for this purpose at the moment, a convention that I think came from the [TS handbook](https://www.typescriptlang.org/docs/handbook/jsdoc-supported-types.html#param-and-returns).

I agree we should prefer `object` in cases where it does make a difference though, assuming we can't be more specific about the object's properties.